staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -157,3 +168,125 @@ func (d *Helper) GetPodsForDeletion(nodeName string) (*podDeleteList, []error) {
 
 	return list, nil
 }
+
+// DeleteOrEvictPods deletes or evicts the pods on the api server
+func (d *Helper) DeleteOrEvictPods(pods []corev1.Pod) error {
```

Comment:
Now that this function is exported, there should be tests specifically calling/exercising this function.

Reply:
Created a test using mocks.